recipes/poco/all/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.5.0)
```

Comment:
I don't think this change is necessary. AFAIC CMake is respecting all minimum versions from all subprojects. This version should only be about the minimum version of this CMake file.

Reply:
Nope. If it is setting as `2.8.12` then `CMP0025` isn't set by default and build with Clang 12.0.0 will failed.

Reply:
This is true the policies are only set once at the root level, I learnt this recently myself.

The version requirements to increase however